qiita_ware/test/test_ebi.py

```diff
@@ -26,15 +25,12 @@
 
 class TestEBISubmission(TestCase):
     def setUp(self):
-        self.path = path.dirname(path.abspath(__file__)) + '/test_data'
+        self.path = path.dirname(path.abspath(__file__)) + \
```

Comment:
you can even use the `partial` function, so it will simplify the following 3 lines.